spec/ParseWebSocketServer.spec.js

```diff
@@ -14,7 +14,9 @@ describe('ParseWebSocketServer', function() {
 
   it('can handle connect event when ws is open', function(done) {
     var onConnectCallback = jasmine.createSpy('onConnectCallback');
-    var parseWebSocketServer = new ParseWebSocketServer({}, onConnectCallback, 5).server;
+    var http = require('http');
+    var server = http.createServer();
+    var parseWebSocketServer = new ParseWebSocketServer(server, onConnectCallback, 5).server;
```

Comment:
needed to add that as the uWS implementation call `.on` when the server is created.